bin/mail/send-to-mailing-list

```diff
@@ -35,7 +35,7 @@ end
 def process_command_line(argv=ARGV)
   {
     csv: nil,
-    limit: 100000,
+    limit: nil,
```

Comment:
I want to avoid calling it 9 times in the send-real-mail script like we've been doing. See https://github.com/code-dot-org/code-dot-org/blob/staging/bin/mail/mailing-1-29-white-house/send-petition-real-mails

The only benefit I can see to breaking it up like that is if it stalls in the middle for some reason, but even then we'd have to do some work to figure out exactly which emails went through and recover gracefully. It's always been an extra hurdle to add these --start --limit calls and it's more places to make a mistake.

I can however leave the default and just call it with a 900000 limit in this script. How does that sound?